edx-platform/pearson-bsg-theme/lms/templates/learner_profile/learner-achievements-fragment.html

```diff
@@ -71,7 +72,7 @@ <h4 class="sr-only">
         % elif own_profile:
             <div class="learner-message">
                 <h4 class="message-header">${_("You haven't earned any certificates yet.")}</h4>
-                % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
+                % if configuration_helpers.get_value('EXPLORE_COURSES_ON_PROFILE', settings.FEATURES.get('COURSES_ARE_BROWSABLE')):
```

Comment:
I think we can make COURSES_ARE_BROWSABLE site aware instead of defining a new setting. @amalbas